audiorecorder/src/main/java/org/odk/collect/audiorecorder/recording/RecordingRepository.kt

```diff
@@ -0,0 +1,22 @@
+package org.odk.collect.audiorecorder.recording
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import java.io.File
+
+class RecordingRepository {
```

Comment:
"Repository" might be the right phrase but this pattern of a singleton that communicates between Services, ViewModels and UI seems pretty standard/nice.